public-lighting-demo-app/web-demo-app/src/main/java/org/opensmartgridplatform/webdemoapp/infra/platform/SoapRequestHelper.java

```diff
@@ -35,6 +36,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SoapRequestHelper.class);
 
+    private static final String ALLOW_ALL_HOSTNAMES = "ALLOW_ALL_HOSTNAMES";
+    private static final String BROWSER_COMPATIBLE_HOSTNAMES = "BROWSER_COMPATIBLE_HOSTNAMES";
```

Comment:
The `web-demo-all` has no dependency on `shared`. Therefore using a couple of string constants instead of the `WebServiceTemplateHostnameVerificationStrategy` enumeration.